client/src/App/index.js

```diff
@@ -15,53 +17,73 @@ import {
 import './style.css';
 
 const App = () => (
-  <div className="App">
-    <Layout>
-      <Switch>
-        <Route
-          path="(/|/home)"
-          exact
-          render={({ history }) => <Home history={history} />}
-        />
-        <Route path="/faq" component={() => <FAQ />} />
-        <Route path="/about-us" component={() => <AboutUs />} />
-        <Route
-          path="/search"
-          component={({ location: { state = {} } }) => (
-            <SearchPage searchState={state} />
-          )}
-        />
-        <Route
-          path="/add-review/:type/:sitspotId"
-          render={({
-            history,
-            match: {
-              params: { type, sitspotId },
-            },
-          }) => (
-            <AddReview
-              history={history}
-              type={type.toUpperCase()}
-              sitspotId={sitspotId}
-            />
-          )}
-        />
-        <Route
-          path="/add-place"
-          component={({ history }) => <AddNewSitSpot history={history} />}
-        />
-        <Route
-          path="/sitspot/:type/:id"
-          render={({
-            history,
-            match: {
-              params: { type, id },
-            },
-          }) => <SinglePlace history={history} type={type} sitspotId={id} />}
-        />
-      </Switch>
-    </Layout>
-  </div>
+  <ImagesContextProvider>
+    <ImagesContext.Consumer>
```

Comment:
no need to wrap the whole app
just the switch, which in it there is all the actual stuff
and you consume it only at the place where u will use the data?
i mean
passing the props like this is not really different
and what's the import weird syntax